### PR TITLE
Eagle 1375

### PR DESCRIPTION
**Pull request by M-Wicenec**
displaying active graph config in the graph name wrapper and in the graph inspector

## Summary by Sourcery

Display the active graph configuration details in the graph inspector and graph name wrapper, enhancing the user interface with better organization and additional functionality.

New Features:
- Display the active graph configuration name and description in the graph inspector and graph name wrapper.

Enhancements:
- Reorder icons in the graph inspector for better organization and user experience.
- Add a new icon in the graph inspector to view the graph configurations table.

### CODE REVIEW

File: src/Eagle.ts [hunk 1]
```diff
@@ -385,6 +385,14 @@ export class Eagle {
         return fileInfo.getText();
     }, this);
 
+    activeConfigText : ko.PureComputed<string> = ko.pureComputed(() => {
```
Comment (james-strauss-uwa) [inline src/Eagle.ts:388]: Can we name this 'activeConfigHtml' to indicate that it returns html

File: src/FileInfo.ts [hunk 2]
```diff
@@ -371,9 +371,9 @@ export class FileInfo {
     getText = () : string => {
```
Comment (james-strauss-uwa) [inline src/FileInfo.ts:371]: Maybe rename to getHtml() here too

File: templates/inspector.html [hunk 3]
```diff
@@ -310,6 +311,22 @@ <h5>Modified Author: </h5>
...
+                            <div class="col-6 col contentObjectTitle">
+                                <h5>Active Config: </h5>
+                            </div>  
+                            <div class="col-6 col contentObjectValue">
+                                <span data-bind="text:$root.logicalGraph().getActiveGraphConfig()?.getName(), eagleTooltip:$root.logicalGraph().getActiveGraphConfig()?.getDescription()"></span>
+                            </div>
+                        </div>
+                        <div class="row inspectorEdgeSrcNodeId contentObject">
+                            <div class="col-6 col contentObjectTitle">
+                                <h5>Config Length: </h5>
```
Comment (james-strauss-uwa) [inline templates/inspector.html:324]: Maybe "Config Fields:" instead of length?

**Review by james-strauss-uwa — APPROVED** [review]
Looks good

File: templates/inspector.html (hunk 3 above)
Comment (M-Wicenec) [reply templates/inspector.html:324]: yea maybe, i wasnt quite sure what to call it without making it too long

**Pull request merged**